### ISSUE TITLE
Duplication of `cascade_duplicates` setting via `DataExtension` will duplicate the duplication process

### ISSUE THREAD

**Issue by mfendeksilverstripe** [dnadesign/silverstripe-elemental#220]
`ElementalArea` currently has following setting:

```
private static $cascade_duplicates = [
    'Elements',
];
```

This is absolutely fine, however in case that I apply the same setting via `DataExtension` on `ElementalArea` the result will be the duplication process running twice which I believe should not be the case.

Example:
```
BlockPage
->LayoutBlock
-->TextBlock
```

After duplication expected:

```
BlockPage
->LayoutBlock
-->TextBlock

BlockPage
->LayoutBlock
-->TextBlock
```

After duplication actual:

```
BlockPage
->LayoutBlock
-->TextBlock

BlockPage
->LayoutBlock
-->TextBlock
-->TextBlock
->LayoutBlock
-->TextBlock
-->TextBlock
```

It's probably not a super critical issue as you shouldn't need to apply the duplication config twice on the same class, but I don't think we should have the ability to run the duplication process twice.

**Comment by robbieaverill** [comment]
Interesting. This seems like it could be a core issue on silverstripe/versioned.

**Comment by robbieaverill** [comment]
Yep this is a core bug, fix at https://github.com/silverstripe/silverstripe-framework/pull/8260 in the 4.1 line

**Comment by robbieaverill** [comment]
Fixed in core (4.1)
